decbinhex4/core/main.gdextension

```diff
@@ -0,0 +1,14 @@
+[configuration]
+
+entry_symbol = "init_library"
+compatibility_minimum = 4.1
+
+[libraries]
+
+linux.debug.x86_64 = "res://core/lib/libcore.so"
+linux.release.x86_64 = "res://core/lib/libcore.so"
+windows.debug.x86_64 = "res://core/lib/libcore.dll"
+windows.release.x86_64 = "res://core/lib/libcore.dll"
```

Comment:
I recently learned that apparently this section works with a more abstract assignment.

```
linux = "res://core/lib/libcore.so"
windows = "res://core/lib/libcore.dll"
```

You may leave the designation of debug or release for future use. Perhaps a specification about CPU architecture is not necessary.